docs/meetings/exeter-jun-2019.md

```diff
@@ -0,0 +1,254 @@
+# Exeter Meeting Notes
+
+June 24-28
+Present HO, DM, MS, OS, SB, BK
+
+- ZenHub - use instead of Trello, as it provides a powerful multi-project view
+  of GitHub. 
```

Comment:
I guess we should start using this. Should discuss in the upcoming VC?

Reply:
Yes, I'm keen to use it.  Can you "invite" everyone, if you haven't already?

Reply:
Funny, I thought I had used Zenhub in Exeter, but tried to log in with GitHub, and it's asking me to grant the permission as if it was the first time. Scary part is that it requests read/write for all public and private repositories :/

Reply:
I guess it needs write permission so that it can automatically create Issues from cards etc.

Reply:
I have now invited all of us in the UK and NZ teams.